source/_posts/2017-05-16-shipit-day-viii-spring-2017.markdown

```diff
@@ -0,0 +1,120 @@
+
+<div align="center">
+{% img /assets/2017-05-16-shipit-day-viii-spring-2017/pic8.png %}
+<sub>Old school advertising for ShipIt Day Demo�s- chalk!</sub>
+</div>
+
+
+<div align="center">
+{% img /assets/2017-05-16-shipit-day-viii-spring-2017/pic9.jpg %}
+<sub>Prize table</sub>
+</div>
+
+ShipIt Day VIII brought to life some incredible projects. Team Freudian Ship, made up of Kate Young, Erin Dorpinghaus, Courtney Ground, Chris Farr and Sean Albery, took first place (and the golden keyboard!) with their project.
+
+>�The Intellectual Property (IP) Analytics team is tasked with managing a data warehouse. This includes ensuring that data updates take place in "jobs." When a job fails it results in stale data and this can impact thousands of associates. This project took advantage of [Amazon Web Services](https://aws.amazon.com/) by connecting tasks associated with managing the database to a voice command [Alexa](https://developer.amazon.com/alexa) skill which results in the ability to manage the database from anywhere the user has an internet connection. To put it in perspective, the database admin will now be able to ensure that the database is up to date without even getting out of bed in the morning. A simple voice command to an [Amazon Echo Dot](https://www.amazon.com/All-New-Amazon-Echo-Dot-Add-Alexa-To-Any-Room/dp/B01DFKC2SO) (which each of us won as a prize) will result in the restarting of failed jobs and even logging JIRA tickets to begin the troubleshooting process if needed.�
+
+ShipIt Judge Dan Stocksick was impressed with the project, �Great use of consumer products into our ecosystem.�
+
+<div align="center">
+{% img /assets/2017-05-16-shipit-day-viii-spring-2017/pic10.png %}
+<sub>Team Freudian Ship with the coveted Gold Keyboard!</sub>
+</div>
+
+Second place went to team Flowalva (Wuchen Wang and Dhruv Patel).
+
+>�Our ShipIt project was a traffic visualization system for web services supporting Millennium. In this project we used Splunk API to collect the data from services and visualize the traffic between nodes in using [Netflix�s vizceral project.](https://github.com/Netflix/vizceral)
+
+Judge Jason Heiting was impressed. �Very neat visualizations.  Intuitive UI for navigation.�
+
+Last but not least, was third place winners, Flying Mongoose (Cihan Kaynak, Matt Stramel, Mahesh Acharya, Taylor Clay).
+
+>�We created a cross platform mobile app which can be used by a pharmacy technician while picking up and scanning medications at the pharmacy to refill [Cerner�s RxStation](https://www.cerner.com/solutions/automated-dispensing-cabinet) cabinets. Prior to this project, an RCP based desktop app was running in a PC mounted on a cart with an attached barcode scanner and label printer. This type of bulky setup was reducing the mobility of the technician.�
```

Comment:
Can apply on previous blocks. The vizceral one only has a single left double-quote.